src/legends/swatches.js

```diff
@@ -93,6 +93,14 @@ function legendItems(scale, {
         --swatchHeight: ${+swatchHeight}px;
       `);
 
+  const palette = label
+      ? swatches.call(div => div.append("p")
```

Comment:
I think we probably don’t want to use a P element here, and instead should have a DIV with more specific margins as needed. Probably slightly less vertical spacing between the label and the swatches than in this current example.

<img width="643" alt="Screen Shot 2022-03-30 at 8 08 45 AM" src="https://user-images.githubusercontent.com/230541/160868380-77a62101-ec4d-45c9-8078-93d1ae20b9f7.png">